src/action/setting.js

```diff
@@ -0,0 +1,22 @@
+import { UNITS } from '../config';
+
+class SettingAction {
+  constructor(store) {
+    this._store = store;
+  }
+
+  //
+  // Bitcoin Unit actions
+  //
+
+  setBitcoinUnit(unit) {
+    if (this._store.settings.unit === unit) {
+      return;
+    } else if (!(unit in UNITS)) {
+      return;
```

Comment:
Probably safe to throw an error here, as it would indicate wrong usage of the api. Maybe best to do this check at the beginning of the function.